### PR TITLE
2017-04-01 - Servicebus and Relay corrected typo

### PR DESCRIPTION
**Pull request by v-Ajnava**
corrected the typo error 

Servicebus : 
microsoft.servicebus/namespaces/topics/subscriptions/rule 
 to microsoft.servicebus/namespaces/topics/subscriptions/rules

Relay: 
microsoft.relay/namespaces/hybridconnection to microsoft.relay/namespaces/hybridconnections
microsoft.relay/namespaces/wcfrelay to microsoft.relay/namespaces/wcfrelays

### CODE REVIEW

File: schemas/2017-04-01/Microsoft.Relay.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
         "type": {
           "type": "string",
           "enum": [
-            "Microsoft.Relay/namespaces/hybridConnection"
+            "Microsoft.Relay/namespaces/hybridConnections"
```
Comment (TianoMS) [inline schemas/2017-04-01/Microsoft.Relay.json]: >hybridConnections [](start = 40, length = 17)

Please update the tests for these changes.

Comment (TianoMS) [inline schemas/2017-04-01/Microsoft.Relay.json]: >hybridConnections [](start = 40, length = 17)

Please update schemas/2015-01-01/deploymentTemplate.json and schemas/2014-04-01-preview/deploymentTemplate.json for these changes.

Comment (v-Ajnava) [reply schemas/2017-04-01/Microsoft.Relay.json]: Have updated the DeplymentTemplate.json

Comment (v-Ajnava) [reply schemas/2017-04-01/Microsoft.Relay.json]: Updated tests

**Comment by TianoMS** [comment]
>                 "$ref": "#/definitions/namespaces_hybridConnection_childResource"

these needs to update as well. 

---
Refers to: schemas/2017-04-01/Microsoft.Relay.json:65 in 4b0b201. [](commit_id = 4b0b201e8d8f2863519b46a1a145e2293c5e2bcb, deletion_comment = False)

**Pull request merged**